packages/wasm-utils/src/lib.rs

```diff
@@ -451,18 +457,36 @@ impl MerkleTree {
 		Uint8Array::from(root.to_bytes().to_vec().as_slice())
 	}
 
-	pub fn create_zk_proof(&self, root: Uint8Array, note: &Note) -> Result<ZkProof, JsValue> {
+	pub fn create_zk_proof(
+		&self,
+		root: Uint8Array,
+		recipient: Uint8Array,
+		relayer: Uint8Array,
```

Comment:
@AhmedKorim for these args, for now, you should pass the current user's public key, as the `recipient` and as the `recipient` too, since the anon node expects that.

https://github.com/webb-tools/anon/blob/40503d19fc3dfdbe6a8689a36d05b721acbbeaec/pallets/mixer/src/lib.rs#L286-L287

Reply:
@drewstone maybe you have a better comment on this, and how to handle this from the dApp for now.

Reply:
yea lets just pass for now the recipient as both the `recipient` and `relayer`. We should be injecting this account directly from PolkadotJS side.